### PR TITLE
murdock: nice compile for better test performance

### PR DESCRIPTION
**Pull request by kfessel**
### Contribution description

Title

### Issues/PRs references

addresses #17894

### CODE REVIEW

**Comment by kaspar030** [comment]
let's see if that has any visible effect on total build times. (the one error so far seems unrelated)

File: tests/pbkdf2/Makefile [hunk 1]
```diff
@@ -8,6 +8,9 @@ USEMODULE += base64
 # Use a terminal that does not introduce extra characters into the stream.
 RIOT_TERMINAL ?= socat
 
+#ensure the rx buffer has some room even with large test patterns
+CFLAGS += -DSTDIO_UART_RX_BUFSIZE=128
```
Comment (kaspar030) [inline tests/pbkdf2/Makefile:12]: (some boards have issues with larger uart transfers, like, the samr21-xpro over it's edbg)

Comment (kfessel) [reply tests/pbkdf2/Makefile:12]: But that isn't an issue of the stdio-buffer-size being to big --i think

Comment (kfessel) [reply tests/pbkdf2/Makefile:12]: In the pbkdf2-test are transfers of upto 67 byte (65+2) only limited by the speed of the serial connection. 
The default rx buffer is 64 bytes -> if the uart isr keep coming in and hence the bytes not processed, it is getting tight in that buffer.

**Comment by kfessel** [comment]
Checking mudock: 
https://ci.riot-os.org/RIOT-OS/RIOT/17918/9bdd7043721e7a0a5004a8fefb2f325e5e6a3937/output/compile/tests/cb_mux_bench/microbit:gnu.txt
--- seems like time off due to emulation
https://ci.riot-os.org/RIOT-OS/RIOT/17918/9bdd7043721e7a0a5004a8fefb2f325e5e6a3937/output/compile/tests/ztimer_mutex_lock_timeout/microbit:gnu.txt
--- may be time off or python timeout

**Comment by kaspar030** [comment]
hm, honestly I would conclude that this doesn't solve the issue of the emulation being inaccurate timing wise. :(

**Comment by kfessel** [comment]
Wo how to still run the test but consider them as warnings?
I had a short look at .murdock but are not sure if there is a concept of warning or how to create them

**Comment by kfessel** [comment]
keep this closed its pointing to the wrong branch

**Pull request closed without merging**